app/controllers/mixins/custom_buttons.rb

```diff
@@ -8,20 +8,23 @@ def custom_toolbar?
   end
 
   def custom_toolbar_explorer?
```

Comment:
Should probably no longer have the question mark in the name, since it returns a nil or a string :)